typeDefs.js

```diff
@@ -227,6 +239,16 @@ const typeDefs = gql`
     buildingNumber: String
     appartment: String
   }
+  input UserForCommentInput {
+    email: String!
+    name: String
+    images: ImageSetInput
+    isAdmin: Boolean
+  }
+  input UserRateInput {
+    user: ID!
```

Comment:
We don't need an UserId here. It is going to be taken from the jwt token inside app

Reply:
I think you can get rid of overall UserRateInput itself

Reply:
Done